test/decl/func/functions.swift

```diff
@@ -123,13 +123,12 @@ func testObjCMethodCurry(_ a : ClassWithObjCMethod) -> (Int) -> () {
 }
 
 // We used to crash on this.
-func rdar16786220(inout let c: Int) -> () { // expected-error {{parameter must not have multiple '__owned', 'inout', '__shared', 'var', or 'let' specifiers}} {{25-29=}}
+func rdar16786220(inout let c: Int) -> () { // expected-warning {{'let' in this position is interpreted as an argument label}} {{25-28=`let`}}
```

Comment:
Let's add a test which would trigger original error and a couple where let is in the middle or last argument/parameter to cover more ground.

Reply:
I added a few where the let is the middle or last parameter in unqualified_name.swift. If I add the following test here as well, would that be sufficient?

```swift
func multipleSpecifiers(inout __owned a: Int) {} // ...
```

Reply:
Yeah, that would be fine, thank you!